bin/build-tests

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
```

Comment:
This builds a bundle with all the tests in it. The reason I chose to do this is so that the tests can be es6 and use the `import` keyword without having to deal with karma's inability to do that.